inc/class-s3-uploads-stream-wrapper.php

```diff
@@ -17,7 +17,35 @@ public static function register( Aws\S3\S3Client $client)
 		static::$client = $client;
 	}
 
+	// Override
+	public function stream_flush() {
+
+		// Add some HTTP headers which can be defined in wp-config.php
+
+		// Expires:
+		if ( defined( 'S3_UPLOADS_EXPIRES' ) ) {
+			$this->params[ 'Expires' ] = S3_UPLOADS_EXPIRES;
+		}
+
+		// Cache-Control:
+		if ( defined( 'S3_UPLOADS_CACHE_CONTROL' ) ) {
+			if ( is_numeric( S3_UPLOADS_CACHE_CONTROL ) ) {
+				$this->params[ 'CacheControl' ] = 'max-age='. S3_UPLOADS_CACHE_CONTROL;
+			} else {
+				$this->params[ 'CacheControl' ] = S3_UPLOADS_CACHE_CONTROL;
+			}
+		}
+
+		// Theses are the parameters passed to S3Client::putObject()
```

Comment:
I haven't looked into it, but is there some more Core Official way of documenting these hooks?